ambry-api/src/main/java/com.github.ambry/router/RouterErrorCode.java

```diff
@@ -84,5 +83,9 @@
   /**
    * The range offsets provided for a getBlob operation are invalid for the specified blob.
    */
-  RangeNotSatisfiable
+  RangeNotSatisfiable,
+  /**
+   * The channel returned to the user in a getBlob operation has been closed before operation completion.
+   */
+  ChannelClosed
```

Comment:
please remember to add these codes to `RestServiceErrorCode::getRestServiceErrorCode()` in your next patch for range requests.